types/magicmirror-module/index.d.ts

```diff
@@ -0,0 +1,91 @@
+// Type definitions for non-npm package magicmirror-module 2.16
+// Project: https://magicmirror.builders/
+// Definitions by: Jalibu <https://github.com/jalibu>
+//                 MichMich <https://github.com/MichMich>
```

Comment:
```suggestion
```
the last bit: this most probably requires MichMich explicit consent here, is that OK to reference him in PR?
I understand MichMich is the owner of original module implementation, but this is the same as owner of types definition (you can always give a reference in implementation code pointing to original idea, etc)

Reply:
I can not mention him here, but I wrote him on Discord. Let's see what happens